### ISSUE TITLE
conversations.history does not have attachments

### ISSUE THREAD

**Issue by WhyIsItSoHardToPickAUsername** [slackapi/bolt-python#980]
(Describe your issue and goal here)

As seen in #978 , conversation.history is supposed to have a 'attachments' property, that will let me know if a file has been uploaded in a message.

I can see attachments property in a parent message if the message is a file uploaded.

```python
{
    "ok": True,
    "oldest": "1698814783.168829",
    "messages": [
        {
            "type": "message",
            "text": "",
            "files": [
                {
                    "id": "F064DM2H656",
                    "created": 1698813725,
                    "timestamp": 1698813725,
                    "name": "myfile",
                    "title": "myfile",
                    "mimetype": "application/octet-stream",
                    "filetype": "binary",
                    "pretty_type": "Binary",
                    "user": "U03CVKELZU6",
                    "user_team": "EDBV1AA7J",
                    "editable": False,
                    "size": 215205,
                    "mode": "hosted",
                    "is_external": False,
                    "external_type": "",
                    "is_public": False,
                    "public_url_shared": False,
                    "display_as_bot": False,
                    "username": "",
                    "url_private": "https://files.slack.com/files-pri/TDBV1AA7J-F064DM2H656/myfile",
                    "url_private_download": "https://files.slack.com/files-pri/myfile",
                    "media_display_type": "unknown",
                    "permalink": "https://example.enterprise.slack.com/files/myfile",
                    "permalink_public": "https://slack-files.com/myfile",
                    "is_starred": False,
                    "has_rich_preview": False,
                    "file_access": "visible",
                }
            ],
            "upload": False,
            "user": "U03CVKELZU6",
            "display_as_bot": False,
            "ts": "1698814783.168829",
            "client_msg_id": "696807b1-414c-4589-a0e2-daf8609379f8",
            "reactions": [{"name": "bob", "users": ["U03CVKELZU6"], "count": 1}],
        }
    ],
    "has_more": False,
    "pin_count": 0,
    "channel_actions_ts": None,
    "channel_actions_count": 0,
}
```

However, in a given thread, if a file is uploaded, conversation.history does not have attachments property.

```python
{
    "ok": True,
    "oldest": "1698815126.385179",
    "messages": [],
    "has_more": False,
    "pin_count": 0,
    "channel_actions_ts": None,
    "channel_actions_count": 0,
}
```

2023-10-31 22:05:30 event is 

```python
{
    "type": "reaction_added",
    "user": "U03CVKELZU6",
    "reaction": "bob",
    "item": {"type": "message", "channel": "C060030E22W", "ts": "1698815126.385179"},
    "item_user": "U03CVKELZU6",
    "event_ts": "1698815129.006300",
}
```

In order to get conversations.history I am using:

```python
result = client.conversations_history(channel=channel_id, inclusive='true', limit=1, oldest=ts)
```
### Reproducible in:

```bash
pip freeze | grep slack
python --version
sw_vers && uname -v # or `ver`
```

#### The `slack_bolt` version

slack-bolt==1.18.0
slack-sdk==3.23.0

(Paste the output of `pip freeze | grep slack`)

#### Python runtime version

(Paste the output of `python --version`)

Python 3.10.0

#### OS info

(Paste the output of `sw_vers && uname -v` on macOS/Linux or `ver` on Windows OS)

ProductName:		macOS
ProductVersion:		14.1
BuildVersion:		23B74
Darwin Kernel Version 23.1.0: Mon Oct  9 21:27:27 PDT 2023; root:xnu-10002.41.9~6/RELEASE_X86_64

#### Steps to reproduce:

(Share the commands to run, source code, and project settings (e.g., setup.py))

1.
2.
3.

### Expected result:

(Tell what you expected to happen)

### Actual result:

(Tell what actually happened with logs, screenshots)

## Requirements

Please read the [Contributing guidelines](https://github.com/slackapi/bolt-python/blob/main/.github/contributing.md) and [Code of Conduct](https://slackhq.github.io/code-of-conduct) before creating this issue or pull request. By submitting, you are agreeing to those rules.

**Comment by seratch** [comment]
Hi @WhyIsItSoHardToPickAUsername, the message with the file attachments is "ts: 1698814783.168829" while the other examples you've shared point to different "ts" ones. Could you double-check if your code is trying to fetch the ones you're expecting?

**Comment by WhyIsItSoHardToPickAUsername** [comment]
The message with the file attachment ("ts: 1698814783.168829") has no thread, it's a single message in a channel. So basically in the channel I uploaded a file, and added a reaction "bob" to the file. 

The 2nd message ("1698815129.006300"), I uploaded a file in an existing thread, and I added a reaction "bob" to the file.  So when I get the "reaction_added" event for this in a thread, I am using conversations.history to try and get the attachment information, but I don't see that. 

Basically, I am trying to implement the solution that was recommended by you in #978  and it's working as long as the reaction_added happen in the main channel, but once the reaction_added happens in a thread, if I use conversations.history to get the message information, it's missing the attachment portion.

What I do is, when I get a reaction_added method for 'bob', I use conversations.history to get that particular message as in:

```python
    result = client.conversations_history(channel=channel_id, inclusive='true', limit=1, oldest=ts)
```

This gives me the attachment information, which I use to download the file, but this information is missing in a thread, it only works in the main channel.

Also, how did you pretty print the JSON? GitHub issue did not pretty print it for me when I opened the issue. I like the pretty print format for JSON that's showing up now. Thank you!
